src/test/scala/ai/privado/languageEngine/java/tagger/collection/CollectionUtilityTest.scala

```diff
@@ -58,6 +58,9 @@ class CollectionUtilityTest extends JavaTaggingTestBase {
       |	public Token login(@RequestBody String somestring) {
       | }
       |
+      | @RequestMapping(method = RequestMethod.GET, value = "/products", produces = "application/json")
```

Comment:
@hiteshbedre Please also add a test case for `@PostMapping("/products")`, I believe above implementation will fail for this case

Reply:
never mind, that is already handled in `none`